src/transformers/models/owlvit/processing_owlvit.py

```diff
@@ -61,6 +61,11 @@ def __call__(self, text=None, images=None, padding="max_length", return_tensors=
                 The image or batch of images to be prepared. Each image can be a PIL image, NumPy array or PyTorch
                 tensor. In case of a NumPy array/PyTorch tensor, each image should be of shape (C, H, W), where C is a
                 number of channels, H and W are image height and width.
+            query_images (`PIL.Image.Image`, `np.ndarray`, `torch.Tensor`, `List[PIL.Image.Image]`, `List[np.ndarray]`,:
+            `List[torch.Tensor]`):
```

Comment:
This needs to stay on one line
```suggestion
            query_images (`PIL.Image.Image`, `np.ndarray`, `torch.Tensor`, `List[PIL.Image.Image]`, `List[np.ndarray]`, `List[torch.Tensor]`):
```